CONTRIBUTING.md

```diff
@@ -165,3 +165,27 @@ general guidelines for the community as a whole:
   respond to an email you are potentially sending to a large number of
   people.  Please consider this before you update.  Also remember that
   nobody likes spam.
+
+## FAQ
+
+**Is it fine to go with kotlin for my contribution?**
```

Comment:
s/kotlin/Kotlin :)
Nothing against Kotline from my side. https://github.com/openhab/openhab-android/issues/428

Reply:
For the reasons given in https://github.com/openhab/openhab-android/issues/428#issuecomment-340157780 I think I'd prefer to continue using Java though, unless there's a compelling argument for using Kotlin in that particular case.
I don't have a strong opinion on Kotlin vs. Java per she, but I really don't like mixing both languages.

Reply:
Hmm, so we should kinda aggree on one of the answers her. Kotlin seems to be growing real quick, but I still see your argument as valid. Java is spread world wide while kotlin probably is only in the Android community.

Reply:
> Java is spread world wide while kotlin probably is only in the Android community.

Exactly. In our specific case the server side is written in Java as well, so it's helpful to use the same language in the client.

Reply:
@mueller-ma Maybe we should revisit this looking at recent news (e.g. https://techcrunch.com/2019/05/07/kotlin-is-now-googles-preferred-language-for-android-app-development/) ... I've toyed around with it for a bit, and initial conversion would be pretty straightforward, with potential of saving about 10% of LoC.